src/WaterPokemon.java

```diff
@@ -0,0 +1,44 @@
+public class WaterPokemon extends Pokemon {
+
+    //    Could place an arraylist for all water type pokemons??
+    private boolean effective;
+    private String sound;
```

Comment:
Je hebt nu voor elke subklasse een `effective` en `sound` attribuut gemaakt, maar als elke subklasse dat heeft, dan zou het misschien makkelijker zijn om dat ergens op een centrale plek te definieren, zodat je dat niet 4 keer hoeft te  typen (en 4 keer hoeft aan te passen als je later toch iets anders wilt). 
Misschien is het dan makkelijk om een soort van super klasse te maken waarin je alle gedeelde attributen kun definieren. 
Oh wacht, dat is de Pokemon klasse!

Voor de variabelen in de subklasses had je hier beter iets specifiek voor WaterPokemon kunnen kiezen, zoals "zwemkleding" of "kanOnderwaterAdemen" en voor een FirePokemon iets als "temperatuur", "aantalLitersLava" of "vlamKleur"

Reply:
Haha inderdaad! Dan is die standaard mee gegeven door de super. Vond het erg jammer dat ik er niet nog even verder aan kon sleutelen voor ik het inleverde maar zal zeker gaande weg er in me eigen tijd aan verder werken met me nieuwe kennis